app/questionnaire/questionnaire_schema.py

```diff
@@ -34,6 +34,9 @@ def get_section_ids(self):
     def get_section(self, section_id: str):
         return self._sections_by_id.get(section_id)
 
+    def get_repeating_list_for_section(self, section_id):
+        return self._sections_by_id.get(section_id).get('repeat', {}).get('for_list')
```

Comment:
The same comment as previously, the team should make a choice. If we do EAFP then everything should be doing that even `QuestionnaireSchema` otherwise it would just be EAFP that calls code that does LBYL etc.